src/ui.js

```diff
@@ -22,6 +22,7 @@ export const makeTable = async () => {
   // Here we simply rearrange company fields in the order in which we want to display them in UI
   companies.map(company => {
     const row = [];
+    company[CREATED_AT_FIELD_NAME] = new Date(company[CREATED_AT_FIELD_NAME]).toString().substring(4, 21)
```

Comment:
Unfortunately this is not a robust implementation. The problem here is that every time the date format is changed on backend, your code here will have to change too.

The better way to do it would be to use existing Javascript's Date APIs - check this function here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString

Reply:
Thank you for the feedback, this was a very helpful exercise.

Reply:
Glad you found this helpful! Good job with the latest implementation.